docs-2.0/7.data-security/4.ssl.md

```diff
@@ -0,0 +1,64 @@
+# SSL encryption
+
+Nebula Graph supports transmission with SSL encryption between the client, Graph service, Meta service, and Storage service. This topic describes how to enable SSL encryption.
+
+## Precaution
+
+Enabling SSL encryption will slightly affect performance.
+
+## Parameters
+
+| Parameter | Default value | Description |
+| ------------------ | ------- | ------------------------------|
+| `cert_path`        | -       | The path to the pem certification. |
+| `key_path`         | -       | The path to the key certification. |
+| `password_path`    | -       | The path to the password file certification. |
+| `ca_path`          | -       | The path to the trusted CA file. |
+| `enable_ssl`       | `false`   | Whether to enable SSL encryption. |
+| `enable_graph_ssl` | `false`   | Whether to enable SSL encryption in Graph service only. |
+| `enable_meta_ssl`  | `false`   | Whether to enable SSL encryption in Meta service only. |
+
+## Certificate modes
+
+To use SSL encryption, SSL certificates are required. Nebula Graph supports two certificate modes.
+
+- Self-signed certificate mode
+
+  In this mode, users need to make the signed certificate by themselves and set `cert_path`, `key_path`, and `password_path` in the corresponding file according to encryption policies.
```

Comment:
just a question
when do we use users instead of you? I saw many “you” with the same meaning of users existing in previous docs.

Reply:
it's about document style. no final conclusion yet.